src/components/Scanner.js

```diff
@@ -18,16 +21,16 @@ export default React.createClass({
                 constraints: {
                     width: 640,
                     height: 480,
-                    facing: "environment" // or user
+                    facingMode: "environment", // or user
                 }
             },
             locator: {
                 patchSize: "medium",
                 halfSample: true
             },
-            numOfWorkers: 2,
+            numOfWorkers: 1,
             decoder: {
-                readers : [ "code_128_reader"]
+                readers : ["ean_reader"]
```

Comment:
perhaps if we're going to change the default reader, we should probably include a few more common ones? personally i use upc and ean for scanning products and isbn codes, seems there's a good chunk of people who would probably be server by having default be upc, ean, code_128 ?